classes/class-nets-easy-logger.php

```diff
@@ -101,7 +101,7 @@ public static function format_log( $checkout_id, $method, $title, $request_args,
 				'body' => $response['body'],
 				'code' => $code,
 			),
-			'timestamp'      => date( 'Y-m-d H:i:s' ), // phpcs:ignore WordPress.DateTime.RestrictedFunctions -- Date is not used for display.
+			'timestamp'      => gmdate( 'Y-m-d H:i:s' ), // phpcs:ignore WordPress.DateTime.RestrictedFunctions -- Date is not used for display.
```

Comment:
This will change it to always use the GMT time over the actual local time for the server. Is that intended @NiklasHogefjord?